index.js

```diff
@@ -0,0 +1,50 @@
+
+function validateLineup(lineup) {
```

Comment:
Functions should do one thing. This function is handling all pieces of this problem, most of this should be broken out into its own smaller functions.